### ISSUE TITLE
Deploy backend and forntend over local internet

### ISSUE THREAD

**Issue by fabiodrbarros** [idurar/idurar-erp-crm#902]
Hello, first of all I'd like to thank you for the project! It's really great.

But I have a question. Is it possible to deploy the backend and frontend within the same internet network? If so, could you give me some advice?

Thanks

**Comment by jumbowalk69** [comment]
@fabiodrbarros yeah its possible to deploy both frontend and backend on same internet. All you need is a static IP for hosting the frontend and you can create any port for hosting the backend.
<b>Example:</b> 
frontend address: 127.0.0.1
backend address: 127.0.0.1:8000

**Comment by Bukunmi-sketch** [comment]
Sorry are u the frontend developer working on the camcam website ?

**Comment by fabiodrbarros** [comment]
@jumbowalk69, but how? sorry i'm new on this type of application
